drivers/flash/nrf_qspi_nor.c

```diff
@@ -564,7 +576,26 @@ static int qspi_nrfx_configure(const struct device *dev)
 		}
 	}
 
-	return 0;
+	if (INST_0_4BA != 0) {
+		if (INST_0_4BA & 0x02) {
+			qspi_nor_write_protection_set(dev, false);
+		}
+
+		struct qspi_cmd cmd = {
+			.op_code = SPI_NOR_CMD_4BA,
+			.rx_buf = NULL,
+			.tx_buf = NULL,
+		};
+		ret = qspi_send_cmd(dev, &cmd, false);
```

Comment:
How about just:
```
		struct qspi_cmd cmd = {
			.op_code = SPI_NOR_CMD_4BA,
		};
		ret = qspi_send_cmd(dev, &cmd, (INST_0_4BA & 0x02));
```
?

Reply:
and right, rest of struct members will be zeroed automatically.